src/utils/utils.test.ts

```diff
@@ -1,22 +1,16 @@
+import { getFileMimeType, getRelativeFilePath, getRootDirectory } from "./file";
```

Comment:
We should also split this test file out so we have `url.ts` and `url.test.ts` for example.

Reply:
Split out tests for each file 👍